Sell/Order/OrdersGettable.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  OrdersGettable.swift
+//  RealifeTech
+//
+//  Created by Olivier Butler on 03/06/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import Foundation
+
+public protocol OrdersGettable {
+    func getOrders(
+        pageSize: Int,
+        page: Int,
+        completion: @escaping (Result<[ApolloType.FragmentOrder], Error>) -> Void)
```

Comment:
I don't think we should return the ApolloType's since they publicly expose a bunch of Apollo specific stuff. What is the correct approach here?

Reply:
I agree with you that we should not expose the Apollo type. Probably we can return an Order object here?

Reply:
Yeah, I think we should return non-apollo objects here, it would give us a good opportunity to move more files out of frontier :)

Reply:
I'mm make these changes today.

Reply:
In line with Mickey's comment below, I'll wait for us to have some architecture specified 👍